### ISSUE TITLE
Do not stop on Latex warnings

### ISSUE THREAD

**Issue by renozao** [craigcitro/r-travis#177]
With the default `warnings_are_errors: true`, warnings thrown by Latex when compiling the manual are treated as errors. 
Is there a way to skip those ones and only consider actual check warnings as errors?

**Comment by craigcitro** [comment]
Adding @jimhester for the latest (since it looks like you're using `language: r` support in Travis).

**Comment by jimhester** [comment]
Can you link to the build where this error occured?

**Comment by renozao** [comment]
I realized I did not use the right place to post this issue. I thought that Travis built-in support was now the only way to go and that r-travis was responsible for the new developments and issues. I see that this generated an issue template to be written. Glad that this mistake will serve other users and r-travis ;)

The good news is that re-looking at the check log, it seems that the error came from actual R check warnings. I was mislead by the final R check log line "`Found warnings, treating as errors (as requested).`", which is preceded by a bunch of latex warnings.
Build is: https://travis-ci.org/renozao/pkgmaker/jobs/111984391
